mgmt/mgmt-core/src/main/java/deployment/mgmt/atrifacts/strategies/classpathfile/JarClasspathReaderSelector.java

```diff
@@ -0,0 +1,7 @@
+package deployment.mgmt.atrifacts.strategies.classpathfile;
+
+import java.io.File;
+
+public interface JarClasspathReaderSelector {
```

Comment:
remove this interface, selector can implement JarClasspathReader

Reply:
done